src/components/containers/SingleArticle.jsx

```diff
@@ -34,6 +37,10 @@ class SingleArticle extends Component {
     if (!isLoading) this.setState(() => ({ modalIsOpen: false }));
   };
 
+  UserFollow = (username, buttonAction) => {
```

Comment:
This function name is meant to start with lowercase letter and it is also confusing to have another variable named `userFollow` on line 95.